form.yml.erb

```diff
@@ -10,6 +10,7 @@ cluster:
 form:
   - account
   - jupyterlab_switch
+  - use_extension
```

Comment:
I'm not sure if we want to add yet another form option. Why not just enable it for all jupyterlab?

Reply:
Now it is enabled for everyone.